src/bidiMapper/domains/script/realm.ts

```diff
@@ -27,9 +27,45 @@ export enum RealmType {
 
 const scriptEvaluator = new ScriptEvaluator();
 
-export class Realm {
-  static readonly #realmMap: Map<string, Realm> = new Map();
+const realmMap: Map<string, Realm> = new Map();
```

Comment:
I thought I wrote a comment by I didn't: let's create a RealmStorage class to hold this map. The instance of that class should be instantiated on the BidiServer and passed around to other classes.